telemetry/definitions/commonDefinitions.json

```diff
@@ -3309,6 +3309,14 @@
                 {
                     "type": "credentialSourceId",
                     "required": false
+                },
+                {
+                    "type": "isStartedFrom",
```

Comment:
If you want a specific enum, you'll have to define it in types. Here, you can probably use the type `source`, which is an open ended string. Then in the Toolkit, we use specific constant values. This keeps the metadata field open ended to support future use cases as well.

I see you have source immediately below this. You might ne able to remove this field.
How would source and isStartedFrom differ?

Reply:
You're correct, this is a redundant field. We've removed it in a more recent commit.